content/update-checker/modules/ROOT/pages/index.adoc

```diff
@@ -0,0 +1,103 @@
+
+== What does the update checker do?
+
+**1. Identify updates and vulnerabilities:**
+The AxonIQ update checker will retrieve available updates and known vulnerabilities for the Axon modules in use.
+This will then be communicated to the Axon Framework application's logs.
+You can see such a sample down below.
+
+[source,terminaloutput]
+----
+13:52:29.074 [main] INFO UpdateChecker -- Your AxonIQ libraries will be checked for updates periodically. See https://axoniq.io/update-check for more information.
+13:52:31.054 [virtual-48] INFO UpdateChecker -- AxonIQ has found the following dependency upgrade(s):
+13:52:31.056 [virtual-48] INFO UpdateChecker -- io.axoniq.console:console-framework-client-spring-boot-starter .. 2.1.0 -> 2.1.5
+13:52:31.057 [virtual-48] INFO UpdateChecker -- io.axoniq.console:console-framework-client-api .................. 2.1.0 -> 2.1.5
+13:52:31.057 [virtual-48] INFO UpdateChecker -- io.axoniq.console:console-framework-client ...................... 2.1.0 -> 2.1.5
+13:52:31.057 [virtual-48] INFO UpdateChecker -- org.axonframework:axon-configuration ............................ 5.0.1 -> 5.0.3
+13:52:31.057 [virtual-48] INFO UpdateChecker -- org.axonframework:axon-eventsourcing ............................ 5.0.1 -> 5.0.3
+13:52:31.057 [virtual-48] INFO UpdateChecker -- org.axonframework:axon-disruptor ................................ 5.0.1 -> 5.0.3
+13:52:31.057 [virtual-48] INFO UpdateChecker -- org.axonframework:axon-modelling ................................ 5.0.1 -> 5.0.3
+13:52:31.057 [virtual-48] INFO UpdateChecker -- org.axonframework:axon-messaging ................................ 5.0.1 -> 5.0.3
+13:52:31.057 [virtual-48] INFO UpdateChecker -- No vulnerabilities have been found in your Axon libraries.
+----
+
+**2. Collect anonymous data**:
+To detect updates and vulnerabilities, the update checker collects anonymous data about your Axon Framework installation.
+This data is sent to AxonIQ and includes technical information about your environment, such as:
+
+- Environment details (OS, Java version)
+- Axon module names and versions in use
+- UUID stored in the user's home directory to identify the installation
+
+.Details of the collected data
+[%collapsible]
+====
+To be exact, the following information is sent to AxonIQ to find updates and vulnerabilities:
+
+- Machine identifier: A UUID stored in the user's home directory, which is used to identify the installation.
+- Session identifier: A UUID generated for each JVM, to identify unique instances.
+- Java information: Version, vendor and architecture.
+- Kotlin information: Version (if applicable).
+- Operating system information: Name, version, and architecture.
+- List of artifacts: Maven coordinates of `io.axoniq` and `org.axonframework` modules in use, including their versions. We never send information about any third-party libraries or dependencies.
+====
+
+[NOTE]
```

Comment:
Question: is the use of `NOTE` enough "in your face"? I want it to be abundantly clear that we don't check any personal/business-specific information whatsoever.